src/meta/processors/jobMan/JobUtils.cpp

```diff
@@ -0,0 +1,72 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef META_JOBUTILS_H_
+#define META_JOBUTILS_H_
+
+#include "meta/processors/jobMan/JobUtils.h"
+
+namespace nebula {
+namespace meta {
+
+/*
+ * It it the key to describe the current job id.
+ * each time a job added, this value will +1
+ * *important* *important* *important*
+ * this key is carefully designed to be the length of jobKey + size(int32_t) + 1
+ * */
+const std::string kCurrJobkey      = "__job_mgr____id"; // NOLINT
```

Comment:
If we use autoIncrementID,  do we still need it?

Reply:
functionly, autoincrementId() is ok. The only reason I prefer use another key, is that 
autoincrementId share the result whit createSpace, createtag...
Which means it will leave holes in returned job id. I just 
thought it is a little uncomportable if a user run one job, which return job id =100, then
he run another job immediately, which return job id=120. 
I'm ok to use autoincrementId(), WDYT.